CHANGELOG.md

```diff
@@ -0,0 +1,27 @@
+# Changelog
+
+## 1.1.3 (2023-12-04)
+
+Full Changelog: [v1.1.3]()
```

Comment:
It would be better to have separate sections for typescript and python

Reply:
Or at least call out version numbers for latest typescript and python covered by the changelog

Reply:
Any good examples we can reference for this? I can't seem to find any monorepos with multiple different versioned libraries.

Reply:
Updated with latest version numbers at the top.